Packs/CyberTriage/Integrations/CyberTriage/CyberTriage.py

```diff
@@ -0,0 +1,192 @@
+import os
+import traceback
+
+import demistomock as demisto  # noqa: F401
+import requests
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable warning for insecure requests when cert validation is disabled
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Get dict of CT instance parameters and command args
+CT_PARAMS = demisto.params()
+CMD_ARGS = demisto.args()
+
+SCAN_OPTIONS = ['pr', 'nw', 'nc', 'st', 'sc', 'ru', 'co', 'lo', 'ns', 'wb', 'fs']
+CT_SERVER = CT_PARAMS['server']
+REST_PORT = CT_PARAMS['rest_port']
+API_KEY = CT_PARAMS['api_key']
+USER = CT_PARAMS['credentials']['identifier']
+PASSWORD = CT_PARAMS['credentials']['password']
+VERIFY_SERVER_CERT = False
+REQ_HEADERS = {'restApiKey': API_KEY}
+USE_PROXY = CT_PARAMS['use_proxy']
+
+
+def IS_2XX(x):
+    return (x / 100) == 2  # Returns true if status code (int) is 2xx
+
+
+# Delete request proxy environment variables if user does not want to use proxy
+if not USE_PROXY:
+    del os.environ['HTTP_PROXY']
```

Comment:
This proxy env deletion appears numerous times in the code.
I guess it belongs in CommonServerPython maybe?

Reply:
I'll replace it with `handle_proxy` from `CommonServerPython`.